src/content/api/managed-integrations.mdx

```diff
@@ -0,0 +1,263 @@
+import Error from '../../components/Error.astro';
+import Badge from '../../components/Badge.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import CodePanel from '../../components/CodePanel.astro';
+
+A Managed Integration is an [Integration](/api/integrations)  which a third party can control.
+
+## Managed Integration Model
+
+### Attributes
+
+<Properties>
+  <Property name="id" type="string">
+    The Managed Integration's unique identifier.
+  </Property>
+
+  <Property name="type" type="string">
+    The type of Managed Integration. Supported types are `paypal-referral`.
+  </Property>
+
+  <Property name="enabled" type="boolean">
+    Flag indicating whether the Managed Integration should become active or inactive.
+  </Property>
+
+  <Property name="status" type="string">
+    Current status of the Managed Integration. Supported values are `created`, `pending`, `provisioning`, `active`, `inactive` and `archived`.
+  </Property>
+
+  <Property name="externalId" type="string">
+    Field used to reference an id from an external system.
+  </Property>
+
+  <Property name="params" type="object">
+    [Params](#params) depending on the Managed Integration type.
+  </Property>
+
+  <Property name="createdAt" type="timestamp">
+    When the Managed Integration was created.
+  </Property>
+
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Managed Integration.
+  </Property>
+
+  <Property name="updatedAt" type="timestamp">
+    When the Managed Integration was updated.
+  </Property>
+
+  <Property name="updatedBy" type="crn">
+    The User or API Key that updated the Managed Integration.
+  </Property>
+
+    <Property name="claimedByAccountId" type="string">
+    Centrapay account id used to claim the Managed Integration.
+  </Property>
+
+  <Property name="claimedBy" type="crn">
+    The User or API Key that claimed the Managed Integration.
+  </Property>
+
+  <Property name="claimedAt" type="timestamp">
+    When the Managed Integration was claimed.
+  </Property>
+
+  <Property name="invitation" type="object">
+    A summary of the associated [Invitation](/api/invitations). See [Invitation Summary](#invitation-summary-model).
+  </Property>
+
+  <Property name="inProgress" type="boolean">
+    Flag indicating whether a status transition is in progress.
+  </Property>
+
+  <Property name="test" type="boolean">
+    A flag which is present if the Managed Integration is for testing
+  </Property>
+</Properties>
+
+---
+
+## Invitation Summary Model
+
+A summary of the [Invitation](/api/invitations) for a Managed Integration.
+
+### Attributes
+
+<Properties>
+  <Property name="id" type="string">
+    The Invitation's unique identifier.
+  </Property>
+
+  <Property name="code" type="string">
+    The Invitation code.
+  </Property>
+
+  <Property name="expiresAt" type="timestamp">
+    When the Invitation expires.
+  </Property>
+</Properties>
+
+---
+
+## Bank Account Model
```

Comment:
@AndyClifford I think this model can be removed.
If you look at the [original PR](https://github.com/centrapay/centrapay-docs/commit/9aa995836b3ac35bd206d331bff48dde5576e29e) it looks like it added a field `managedByAccountId` and this model. That field has since been removed so I think this is irrelevant.